intern/cycles/kernel/kernel_path_state.h

```diff
@@ -65,6 +65,14 @@ ccl_device_inline void path_state_init(KernelGlobals *kg,
     state->volume_stack[0].shader = SHADER_NONE;
   }
 #endif
+  float wavelength_offset = path_state_rng_1D(kg, state, PRNG_WAVELENGTH);
+  float offset_two = fmod(wavelength_offset + (1.0f / 3.0f), 1.0f);
+  float offset_three = fmod(wavelength_offset + (2.0f / 3.0f), 1.0f);
+  state->wavelengths = make_float3(
+    (wavelength_offset * 350) + 380,
+    (offset_two * 350) + 380,
+    (offset_three * 350) + 380
```

Comment:
It might be useful to declare what `350` and `380` mean so there's no misunderstandings as to the intention of the code.

Reply:
Certainly! Once I understand how, these will be passed in at render-time as the low and high wavelengths for sampling, I'm completely against having these 'magic numbers' in here but until I know Cycles and Blender better, this was the quickest way to get something rendering.

Thanks for the feedback :)

Reply:
As stated above, the low and high bounds for the renderer ideally should be passed in as properties on KernelGlobals, which I attempted to do but couldn't find a good place to instantiate them. I haven't looked thoroughly at whether we would need to consider anything extra if sampling a larger or smaller wavelength range, but sampling a larger range is a bit pointless at the moment since the conversion functions (upsampling rgb to wavelength, and wavelength to XYZ) are undefined outside of this range.